neuro_evolution_ctrnn/tools/experiment.py

```diff
@@ -100,12 +107,27 @@ def _setup(self):
         stats.register("avg", np.mean)
         stats.register("std", np.std)
         stats.register("max", np.max)
-        if self.config.use_worker_processes:
+
+        system_cpu_count = os.cpu_count()
+        if self.number_of_workers <= 0 or self.number_of_workers > system_cpu_count:
+            raise RuntimeError(
+                "{} is an incorrect number of processes. Your system only supports {} workers and it must be at least "
```

Comment:
Ja stimmt, ich habe es auf

```python
raise RuntimeError("{} is an incorrect number of workers for your system, because your CPU only supports "
                               "running between 1 and {} processes in parallel."
                               "".format(self.number_of_workers, system_cpu_count))
```
geändert